.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,116 @@
+name: Build (Xcode 13.6)
+
+env:
+  PROJECT_NAME: "PactConsumerSwift.xcodeproj"
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  test_xcodebuild:
+    name: Run tests
+    runs-on: macOS-10.15
```

Comment:
Apparently we can only run `macOS-10.15` or `macOS-11` here on GitHub Actions? I tried with `macOS-10.13` but it failed as there were no runners available. Having macOS 10.15 limits us with the tools we use to support the build as they won't install on macOS 10.15 with Xcode 10.3, stating Xcode is too old. 
The system, simulator and tools versions in this file are the lowest I could find within the reasonable time I had available.